packages/blockchain-wallet-v4-frontend/src/modals/index.tsx

```diff
@@ -96,74 +100,199 @@ const AddBankYodlee = React.lazy(() => import('./Brokerage/Banks/AddBankYodlee')
+        {props.modals.find((modal) => modal.type === ModalName.SEND_ETH_MODAL) ? <SendEth /> : null}
+        {props.modals.find((modal) => modal.type === ModalName.SEND_XLM_MODAL) ? <SendXlm /> : null}
+        {props.modals.find((modal) => modal.type === ModalName.SHOW_BTC_PRIVATE_KEY_MODAL) ? (
+          <ShowBtcPrivateKey />
+        ) : null}
+        {props.modals.find((modal) => modal.type === ModalName.SHOW_USED_ADDRESS_MODAL) ? (
+          <ShowUsedAddresses />
+        ) : null}
+        {props.modals.find((modal) => modal.type === ModalName.SHOW_XPUB_MODAL) ? (
+          <ShowXPub />
+        ) : null}
+        {props.modals.find((modal) => modal.type === ModalName.SIGN_MESSAGE_MODAL) ? (
+          <SignMessage />
+        ) : null}
+        {props.modals.find((modal) => modal.type === ModalName.SIMPLE_BUY_MODAL) ? (
+          <SimpleBuy />
+        ) : null}
+        {props.modals.find((modal) => modal.type === ModalName.SUPPORT_MODAL) ? <Support /> : null}
+        {props.modals.find((modal) => modal.type === ModalName.SWAP_MODAL) ? <Swap /> : null}
+        {props.modals.find((modal) => modal.type === ModalName.TRANSFER_ETH_MODAL) ? (
+          <TransferEth />
+        ) : null}
+        {props.modals.find((modal) => modal.type === ModalName.TRADING_LIMITS_MODAL) ? (
+          <TradingLimits />
+        ) : null}
+        {props.modals.find((modal) => modal.type === ModalName.TWO_STEP_GOOGLE_AUTH_MODAL) ? (
+          <TwoStepGoogleAuthenticator />
+        ) : null}
+        {props.modals.find((modal) => modal.type === ModalName.TWO_STEP_SETUP_MODAL) ? (
+          <TwoStepSetup />
+        ) : null}
+        {props.modals.find((modal) => modal.type === ModalName.TWO_STEP_YUBICO_MODAL) ? (
+          <TwoStepYubico />
+        ) : null}
+        {props.modals.find((modal) => modal.type === ModalName.UPGRADE_ADDRESS_LABELS_MODAL) ? (
+          <UpgradeAddressLabels />
+        ) : null}
+        {props.modals.find((modal) => modal.type === ModalName.UPGRADE_FOR_AIRDROP_MODAL) ? (
+          <UpgradeForAirdrop />
+        ) : null}
+        {props.modals.find((modal) => modal.type === ModalName.INTEREST_PROMO_MODAL) ? (
+          <InterestPromo />
+        ) : null}
+        {props.modals.find((modal) => modal.type === ModalName.VERIFY_MESSAGE_MODAL) ? (
+          <VerifyMessage />
+        ) : null}
+        {props.modals.find((modal) => modal.type === ModalName.WELCOME_MODAL) ? <Welcome /> : null}
+        {props.modals.find((modal) => modal.type === ModalName.CUSTODY_WITHDRAW_MODAL) ? (
+          <Withdraw />
+        ) : null}
+        {props.modals.find(
+          (modal) => modal.type === ModalName.SEND_XLM_CREATE_ACCOUNT_LEARN_MODAL
+        ) ? (
+          <XlmCreateAccountLearn />
+        ) : null}
+        {props.modals.find((modal) => modal.type === ModalName.SEND_XLM_RESERVE_LEARN_MODAL) ? (
+          <XlmReserveLearn />
+        ) : null}
```

Comment:
super cool, I red article that it's better to use ternary and pass null for performance stuff this is obvious case 👏